### PR TITLE
Foundations/the front end/developer tools.md

### PR DESCRIPTION
**Pull request by dylan-workhub**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Added knowledge checks to the developer tools lesson.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#22259

### CODE REVIEW

File: foundations/the_front_end/developer_tools.md [hunk 1]
```diff
@@ -38,3 +38,17 @@ Make sure you can do each of the following once you have finished the assignment
...
 </div>
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you're having trouble answering the questions below on your own, review the material above to find the answer.
+
+* How do you open developer tools in your browser?
+* How do you select a specific element on your page with your browser's developer tools?
+* How do you change CSS in real time on specific elements of a web page with your browser's developer tools?
+* What does a strikethrough in a CSS element mean in your browser's developer tools?
+* How do you check every inhereted style for an element in your browser's developer tools?
```
Comment (rlmoser99) [inline foundations/the_front_end/developer_tools.md:49]: Update spelling to 'inherited'

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Great job on creating this list of questions. It is really thorough and will help make sure students are equipped before they move on to the next lesson. THANK YOU!

File: foundations/the_front_end/developer_tools.md (hunk 1 above)
Comment (dylan-workhub) [reply foundations/the_front_end/developer_tools.md]: Oops! Just updated, thanks for noticing.

**Review by rlmoser99 — APPROVED** [review]
Thanks for making the spelling correction. This looks great!

**Pull request merged**